android/app/src/main/res/values/colors.xml

```diff
@@ -9,6 +9,6 @@
   <color name="errorMessage">#FF0000</color>
   <color name="errorLogButton">#66FFFFFF</color>
   <color name="colorLight">#F6F6F7</color>
-  <color name="splashBackground">#FFFFFF</color>
   <color name="iconBackground">#FFFFFF</color>
+  <color name="splashscreen_background">#FFFFFF</color>
```

Comment:
We used to be using camelCase for the other color names 🤔

Reply:
I don't think that this naming convention of ours is so strict 🤔 
In fact all the resources and official guides suggest doing it the other way: for example [see here](https://developer.android.com/guide/topics/resources/providing-resources).
The other thing is that `expo-splash-screen` module is already designed to work with some well-known assets names and I just propagated it here.